app/policies/company_policy.rb

```diff
@@ -0,0 +1,27 @@
+class CompanyPolicy < ApplicationPolicy
+  DEFAULT_ATTRS = %i[name cnpj address phone active code discount].push(user_ids: [])
+
+  def permitted_attributes
+    DEFAULT_ATTRS
+  end
+
+  def create?
+    user.admin?
+  end
+
+  def update?
+    user.admin?
```

Comment:
Once it's all the same:
```suggestion
    create?
```